velox/functions/prestosql/aggregates/ArbitraryAggregate.cpp

```diff
@@ -266,11 +266,11 @@ exec::AggregateRegistrationResult registerArbitraryAggregate(
           .argumentType("T")
           .build()};
 
-  auto name = prefix + kArbitrary;
+  std::vector<std::string> names = {prefix + kArbitrary, prefix + kAnyValue};
   return exec::registerAggregateFunction(
-      name,
+      names,
       std::move(signatures),
-      [name](
+      [name = names.front()](
```

Comment:
This `name` is used for error message. I passed the first name, also open to pass a concatenated string like "arbitrary/any_value".